res/css/views/rooms/_EventTile.scss

```diff
@@ -65,6 +65,13 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         }
     }
 
+    .mx_ReactionsRow {
+        display: flex;
+        flex-flow: wrap;
+        align-items: center;
+        gap: $spacing-4;
```

Comment:
Since 6px is not available as spacing variable, `$spacing-4` is selected instead.